src/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { InputHTMLAttributes, ReactNode, useState } from 'react';
+import { Typography } from '..';
+import cx from 'classnames';
+import styles from './Checkbox.module.scss';
+import { Colors } from '../../styles';
+
+export interface CheckboxProps extends InputHTMLAttributes<HTMLInputElement> {
+  label?: ReactNode;
+  disabled?: boolean;
+  checked?: boolean;
+  color?: Colors;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => unknown;
+}
+
+function Checkbox({
+  checked,
+  onChange,
+  color = 'primary',
```

Comment:
I don't think we have a color of primary? Did this actually pass type checking?

Reply:
It was the first option intellisense gave me

Reply:
And the type decl is `export type Colors = 'primary' | 'success' | 'error';` unless I'm using an outdated version